### DISCUSSION TITLE
PGNet Inference recognition socre

### DISCUSSION THREAD

**Discussion by agzeroo** [PaddlePaddle/PaddleOCR#12371] [Q&A]
I am testing using PGNet.
It seems that the threshold can be adjusted when making inferences.
However, it seems that only the threshold related to detection is adjusted.
I would like to print the results by adjusting the score or threshold of the character recognition part.

Is there a way?
And I would appreciate it if you could explain the threshold of PGNet.

**Comment by TingquanGao** [comment]
The PGNet is end-to-end ocr model, so there is no rec threshold can be set.

**Comment by agzeroo** [comment]
> The PGNet is end-to-end ocr model, so there is no rec threshold can be set.

Thank you for answer.
So, is it not possible to check rec-related scores in inference?

**Comment by tink2123** [comment]
> > The PGNet is end-to-end ocr model, so there is no rec threshold can be set.
> 
> Thank you for answer. So, is it not possible to check rec-related scores in inference?

Yes
